content/en/continuous_integration/pipelines/circleci.md

```diff
@@ -49,6 +49,9 @@ To set custom tags to all the pipeline and job spans generated by the integratio
 
 `?tags=key1%3Avalue1%2C%22key2%3A+value+with+%2C+comma%22%2Ckey3%3Avalue3`
 
+#### Integrate with Datadog teams
+To display and filter the teams associated with your pipelines, add `team:<your-team>` as a custom tag.
```

Comment:
Let's add a sentence after: 
The custom tag name must exactly match your [Datadog Teams](https://docs.datadoghq.com/account_management/teams/) team handle.

Reply:
Apply this change to all pages

Reply:
```suggestion
To display and filter the teams associated with your pipelines, add `team:<your-team>` as a custom tag. The custom tag name must match your [Datadog Teams][8] team handle exactly.
```